homeassistant/components/smappee/binary_sensor.py

```diff
@@ -0,0 +1,173 @@
+"""Support for monitoring a Smappee appliance binary sensor."""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+
+from .const import BASE, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+BINARY_SENSOR_PREFIX = "Appliance"
+PRESENCE_PREFIX = "Presence"
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the Smappee binary sensor."""
+    smappee_base = hass.data[DOMAIN][BASE]
```

Comment:
Why don't we allow multiple instances ? Since it's OAuth2, someone could link two accounts?

Reply:
We'd like to start with having only a single instance and keep the multiple accounts for another PR.